tests/test_token_manager.py

```
@@ -0,0 +1,451 @@
+    def test_store_token_metadata_with_type(self, mock_github_class):
+        """Test storing token metadata includes token type."""
+        mock_github = Mock()
+        mock_rate_limit = Mock()
+        mock_rate_limit.core.limit = 5000
+        mock_rate_limit.core.remaining = 4999
+        mock_rate_limit.core.reset = datetime.now(timezone.utc)
+        mock_github.get_rate_limit.return_value = mock_rate_limit
+        mock_github_class.return_value = mock_github
+
+        config = ConfigManager()
+        manager = TokenManager(token="ghp_classic_token", config_manager=config)
```

Ruff (0.13.0)

Possible hardcoded password assigned to argument: "token"

(S106)